seeq/addons/correlation/utils/_sdl.py

```diff
@@ -71,3 +72,40 @@ def get_workbook_worksheet_workstep_ids(url):
     if 'workstepId' in params:
         workstep_id = params['workstepId'][0]
     return workbook_id, worksheet_id, workstep_id
+
+
+def is_ipv4(string):
+    temp_str = None
+    try:
```

Comment:
I think theses nested try/excepts can be simplified. 


There is already a `parse_url` function in `_sdl.py`. You could do something like this:

```html
parsed_url = parse_url(string)
try:
    ipaddress.IPv4Network(parsed_url.hostname)
	return True
except ValueError:
    return False
```